site/index.css

```diff
@@ -43,10 +43,12 @@ body {
   float: right;
   color: #FFFAFB;
   font-family: "Palanquin Dark";
+  font-size: 1.2em;
```

Comment:
Just realized that I asked you to make this bigger :sweat_smile: 

What resolution are you using to test this? I'm guessing the header and subheader are getting too small because your width is less than 620px (thus the media query thinks it's a mobile phone in portrait orientation). If that's the case, try testing with some common device dimensions instead. Firefox's [responsive design mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mode/index.html) has good options for this, for example

Reply:
I use Chrome developer tool and adjust the width to 620px and less, maybe some device has bigger width so it doesn't work with them, I can try to figure the general width size.